src/components/structures/TimelinePanel.tsx

```diff
@@ -484,17 +485,17 @@ class TimelinePanel extends React.Component<IProps, IState> {
 
         logger.debug(
             `TimelinePanel(${this.context.timelineRenderingType}): Debugging info for ${room?.roomId}\n` +
-                `\tevents(${eventIdList.length})=${JSON.stringify(eventIdList)}\n` +
-                `\trenderedEventIds(${renderedEventIds?.length ?? 0})=` +
-                `${JSON.stringify(renderedEventIds)}\n` +
-                `\tserializedEventIdsFromTimelineSets=${JSON.stringify(serializedEventIdsFromTimelineSets)}\n` +
-                `\tserializedEventIdsFromThreadsTimelineSets=` +
-                `${JSON.stringify(serializedEventIdsFromThreadsTimelineSets)}\n` +
-                `\tserializedThreadsMap=${JSON.stringify(serializedThreadsMap)}\n` +
-                `\ttimelineWindowEventIds(${timelineWindowEventIds.length})=${JSON.stringify(
-                    timelineWindowEventIds,
-                )}\n` +
-                `\tpendingEventIds(${pendingEventIds.length})=${JSON.stringify(pendingEventIds)}`,
+            `\tevents(${eventIdList.length})=${JSON.stringify(eventIdList)}\n` +
```

Comment:
Normally, the code is already formatted by Prettier. Please do not change the formatting :)